src/components/question/question.js

```diff
@@ -0,0 +1,21 @@
+import './question.css'
```

Comment:
you can rename file from `question.js` to `index.js` to import question from `src/components/question`